packages/bpk-docs/src/components/DocsPageBuilder/TokenSwitcher.js

```diff
@@ -119,12 +119,14 @@ TokenSwitcher.defaultProps = {
 export default TokenSwitcher;
 
 export const connect = (tokenSwitcher, table) => {
-  /* eslint-disable react/no-multi-comp */
+  /* eslint-disable react/no-multi-comp, react/no-this-in-sfc */
   class SwitchAwareTable extends Component {
     constructor(props) {
       super(props);
+
       this.state = { platform: null };
       this.onSwitch = platform => this.setState({ platform });
+
```

Comment:
I reckon we could refactor this to not use `this` (and then won't need the opt out) fairly easily - did you try that already though?

Reply:
This PR is big enough :)